README.md

```diff
@@ -1,16 +1,88 @@
-# Title
+# Building Cells: Getter and Setter Methods
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Builds a setter method to calculate computed properties
 
 ## Introduction
 
-## SWBAT 1
+We've seen so far that we can write methods inside our classes
+that allow us to access and change instance variables:
 
-## SWBAT 2
+```js
+speak() {
+return `${this.name} says 'woof!'`
+}
+
+haveBirthday() {
+this.age += 1
+}
+```
+
+These types of methods work just fine in most cases. There is,
+however, additional JS syntax that we can use: `get` and `set`. `get` is
+used to identify methods that return calculated or dynamic data based an
+object's attributes. `set` is used for methods where we want to change an
+attribute in a controlled way. In this lab, we will be building on the
+concept of _instance variables_ to illustrate these methods.
+
+
+## Builds a Setter Method to Calculate Computed Properties
+
+Let's look at an example that may be familiar:
+
+```js
+class Bird {
+  constructor(name) {
+    this.name = name;
+  }
+
+  set phrase() {
+    this.phrase = phrase;
+  }
+
+  get speak() {
+    return `${this.name} says ${this.phrase || 'squawk'}`
+  }
+}
+```
+Our class of `Bird` accepts the parameter of `name`, which is set each time a new
+instance of `Bird` is created. When `phrase` is set, our new `Bird` instance
+can speak a phrase. If it is not set, it will squawk.
+
+```js
+var buddy = Bird.new("Buddy")
+
+buddy.phrase = "What'cha doin'?";
+buddy.speak() // returns 'Buddy says What'cha doin'?'
+```
+
+Building on this concept, we're going to build our own shape calculator!
+- First, let's create a class of `Circle`
+- `Circle` will accept 1 parameter of `radius` as instance data
+- Define a `constant` for `pi` as `3.14`
```

Comment:
The first test mentions getting the radius - I rearranged the test structure to emphasize that this.radius is the only _real_ property, the one we assign at the beginning, while everything else is calculated on demand. maybe it is worth adding some message here recognizing this distinction?